### ISSUE TITLE
Classifing commits

### ISSUE THREAD

**Issue by niloofartrg** [SpoonLabs/coming#166]
Hello,
For classifying commits I need a good amount of labeled commits, and as I understand there are no labels for the DRR commits (except for the tool in the name), so how should I go about it? Thanks. @monperrus

**Comment by monperrus** [comment]
indeed, the tool name is the label.

**Comment by niloofartrg** [comment]
but what about the other possibilities? So I just know one of the possible labels.

**Comment by monperrus** [comment]
yes, this is the only ground truth we have.

maybe some patches are identical, which means that they can be merged with multiple labels.

**Comment by niloofartrg** [comment]
still, the amount of data in each repair tool is small, for example, there are thirteen file-pairs for Arja, I don't think the results of SVM would be reliable in such a case. Am I right? @monperrus

**Comment by niloofartrg** [comment]
should I predict whether a commit can be generated by a specific repair tool or not, or can it be generated by any repair tool or not? @monperrus

**Comment by niloofartrg** [comment]
In more details, now I have generated features for all the commits in the DRR correct folder. 
So, now I have to fill in a binary table (0,1) with rows corresponding to the commits and the columns corresponding to the repair tools in order to do the ML phase.

My question is that How I should fill in this table? 
I can fill the spots where the name of the repair tool is in the file and the identical files, but what about the rest of the table ? should I use the results from the coming project? @monperrus

**Comment by monperrus** [comment]
> should I predict whether a commit can be generated by a specific repair tool or not

yes, and we would have N classifers where n is the number of repair tools

**Comment by monperrus** [comment]
We'll use the RepairThemAll data once the pipeline is running.
